zerver/openapi/zulip.yaml

```diff
@@ -17562,6 +17563,21 @@ components:
           - type: integer
       example: "20"
       required: false
+    CanRemoveSubscribersGroupId:
+      name: can_remove_subscribers_group_id
+      in: query
+      description: |
+        ID of the user group whose members are allowed to unsubscribe others
+        from the stream.
+
+        This setting can only be set to system user groups except `@role:internet`
+        and `@role:owners` group.
+
+        **Changes**: New in Zulip 7.0 (feature level 161).
```

Comment:
I expanded this to explain the exact model for what this does. https://github.com/zulip/zulip/pull/23992#issuecomment-1418250055

There's a bit of duplication between this and this block, which I didn't fix; probably that needs a similar adjustment to explain what it actually does (or reference the main "unsubscriber users" API endpoint description); can you do a commit for that in a follow-up PR?

```
 can_remove_subscribers_group_id:                                         
   type: integer                                                          
   description: |                                                         
     ID of the user group whose members are allowed to unsubscribe others 
     from the stream.                                                     
                                                                          
     **Changes**: New in Zulip 6.0 (feature level 142).                   
```